node/service/chain-specs/example.json

```diff
@@ -108,6 +138,14 @@
       "authorityDiscovery": {
         "keys": []
       },
+      "society": {
```

Comment:
The `society"` pallet was removed previously. You should remove it from the example.json file.